src/main/java/io/botcrafting/botcraft/core/handler/chain/MessageChain.java

```diff
@@ -0,0 +1,27 @@
+package io.botcrafting.botcraft.core.handler.chain;
```

Comment:
we probably will not have other chains inside handler, I think it would be clear if we had a message package inside core (botcraft.core.message.handler) and this class would be inside handler package, also I don't know if handler is clear enough, but just commenting for the record, not needed change.